jo-rdan/prop-lite
# [feature #167001410] finishes endpoint for signup

# What does this PR do?
It creates a signup user endpoint.

## Description of Task to be done?

* It specifies the user registration route in the app.js file located in the server directory
* It creates a route for user signup.
* It creates a user registration control controlling all actions of the user.
* It creates data structure arrays for storing user information.

### How should this be manually tested?
Start the Server and POSTMAN.
Browse the following endpoint route: localhost:5000/api/v1/auth/signup.
Submit the required information with POST method.
See if all of the validations work once you don't submit any of the required information.
After passing the validations, you should receive a 201 HTTP Status Code along with the information that you have successfully submitted.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167001410
